.github/workflows/ci.yml

```diff
@@ -812,49 +815,69 @@ jobs:
           name: windows-v22.2.0-local.xml
           path: ./windows_local.xml
 
-
-  release:
-    name: "Release"
-    if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
-    needs: [smoke-tests, docs-build, build-test, build-test-ubuntu, build-test-ubuntu-minimal]
+  package:
+    name: Package library
+    needs: [build-test, build-test-ubuntu, build-test-ubuntu-minimal, docs-build]
     runs-on: ubuntu-latest
     steps:
-      - name: Set up Python
-        uses: actions/setup-python@v5
+      - name: Build library source and wheel artifacts
+        uses: ansys/actions/build-library@v6
         with:
-          python-version: 3.9
+          library-name: ${{ env.PACKAGE_NAME }}
+          python-version: ${{ env.MAIN_PYTHON_VERSION }}
+
+
+  # release-bis:
+  #   name: "Release"
+  #   if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
+  #   needs: [docs-build, build-test, build-test-ubuntu, build-test-ubuntu-minimal]
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - name: Set up Python

```

Comment:
Please delete dead code. You shouldn't have to go back to this stage. And you can always find the commit otherwise...

Reply:
Sure, I wanted to make sure I hadn't forgot any step that was already implemented. I will remove it now.